app/views/annotations/_annotation_list.html.erb

```diff
@@ -1,9 +1,17 @@
 <ul class='annotation_text_list'>
-  <% if annotation_category.annotation_texts.empty? %>
+  <%
+    text = []
+    annotation_category.annotation_texts.each do |annotation_text|
+      if (annotation_text.user.nil? || (annotation_text.user.admin? || annotation_text.user == @current_user))
```

Comment:
The user is nil when the category/annotation is autogenerated when building the database, so I assume all those should be accessible to everyone.